addons/modules/fnc_moduleAttack.sqf

```diff
@@ -0,0 +1,62 @@
+/* ----------------------------------------------------------------------------
+Function: CBA_fnc_moduleAttack
+
+Description:
+    A function for commanding a group to attack a location with information
+    parsed from a module.
+
+Parameters:
+    - Group (Array of Classnames or an array of a config)
```

Comment:
That was left over from when I was still trying to figure out a way to spawn groups through an array of class names or a config. The idea was to pass it as [configfile,"CfgVehicles","C_man_1"] instead of the normal config style. I am working on the headers and the other fixes you've pointed out.